### PR TITLE
adding .Tex version of hands-out file

### PR DESCRIPTION
**Pull request by Degoot-AM**
This PR adds `.Tex` version of hands-out file for better edit suggestions and reveiw.

### CODE REVIEW

File: learners/files/hands-on.tex [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    \item \( \gamma \): Recovery rate
+    \item \( \mu \): Death rate (natural death rate)
+    \item \( N \): Total population size, \( N = S + E + I + R \).
+\end{itemize}
+
+ The parameter $\beta$ is derived from the multiplication of $p$
+  and $c$, where $p$ is the probability of transmission during contact, and $c$ 
+  is the contact rate, defined as the average number of contacts per unit of time. 
+
+\section{Equations}\label{eqs}
```
Comment (avallecam) [inline learners/files/hands-on.tex:107]:
```suggestion
Model parameters are often (but not always) specified as rates.
The rate at which an event occurs is the inverse of the average time until that event.
For example, in the SEIR model, the recovery rate $\gamma$ is the inverse of the average infectious period.\\

Values of these rates can be determined from the natural history of the disease.
For example,  if people are on average infectious for 8 days, then in the model, 
1/8 of currently infectious people would recover each day
(i.e. the rate of recovery, $\gamma=1/8=0.125$).

\section{Equations}\label{eqs}
```

File: learners/files/hands-on.tex [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    \item \( \beta \): Transmission rate
+    \item \( \sigma \): Rate of progression from exposed to infectious
+    \item \( \gamma \): Recovery rate
+    \item \( \mu \): Death rate (natural death rate)
+    \item \( N \): Total population size, \( N = S + E + I + R \).
+\end{itemize}
+
+ The parameter $\beta$ is derived from the multiplication of $p$
+  and $c$, where $p$ is the probability of transmission during contact, and $c$ 
+  is the contact rate, defined as the average number of contacts per unit of time.
```
Comment (avallecam) [inline learners/files/hands-on.tex:105]:
```suggestion
  is the contact rate, defined as the average number of contacts per unit of time.\\
```

File: learners/files/hands-on.tex [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+\usepackage{amsmath}
+\author{Andree Valle Campos and Abdoelnaser M Degoot \\ Epiverse-TRACE Team @ LSHTM }
+\title{Simple Introduction to Mathematical Modelling of Infectious Diseases}
+\begin{document}
+\maketitle
+
+\section{Intrdocution}
+This document aims to assess  your understanding of the fundamental 
+principles of mathematical modeling while guiding you in constructing models using 
+a simple SEIR framework for infectious disease outbreaks.
```
Comment (avallecam) [inline learners/files/hands-on.tex:19]:
```suggestion
a simple SEIR framework for infectious disease outbreaks.\\
```

**Review by avallecam — CHANGES_REQUESTED** [review]
Thanks @Degoot-AM for working on this! This totally fits the aims of the session to assess the concepts learned before working in code.

I added some minor edits to increase the space length as some responses are expected to write more words. 

I proposed two major addition: 

- Add some paragraphs about rates and how they are calculated from time period. Also, pasted adapted paragraphs to lead learners read how the R0 is related with beta and gamma.

- List of parameters as sentences, and table of parameters to fill. This practical aims to write by hand the numbers before working on code in the next session.

**Comment by Degoot-AM** [comment]
> Thanks @Degoot-AM for working on this! This totally fits the aims of the session to assess the concepts learned before working in code.
> 
> I added some minor edits to increase the space length as some responses are expected to write more words.
> 
> I proposed two major addition:
> 
>     * Add some paragraphs about rates and how they are calculated from time period. Also, pasted adapted paragraphs to lead learners read how the R0 is related with beta and gamma.
> 
>     * List of parameters as sentences, and table of parameters to fill. This practical aims to write by hand the numbers before working on code in the next session.

Thank you @avallecam for your valuable comments

**Pull request merged**